command.go

```diff
@@ -53,8 +53,10 @@ func NewCommand(
 		},
 	}
 
-	cmdListTests.Flags().StringSliceP("id", "i", nil, "Only run tests with the given identifier")
-	cmdListTests.Flags().StringSliceP("tag", "t", nil, "Only run tests with the given tags")
+	cmdListTests.Flags().BoolP("only-suites", "s", false, "Only list suites")
+	cmdListTests.Flags().StringSliceP("suite", "Z", nil, "Only list suites specified")
```

Comment:
Maybe this "Only list suites specified" should be "Only list suite and contained tests" or something along those lines? Since if the only-suites flag is not set it will behave as before where the suite and tests are printed out. Also might be more in line with the other descriptions on line 58, 59.

Reply:
updated to `Only list specified suites and their contained tests`